Demo/Demo/Extensions/UIViewController+Extension.swift

```diff
@@ -9,4 +9,11 @@ extension UIViewController {
             viewController.removeFromParent()
         }
     }
+
+    func label(_ text: String) -> UILabel {
```

Comment:
```suggestion
    func makeLabel(_ text: String) -> UILabel {
```
To align with [Swift Naming Conventions](https://www.swift.org/documentation/api-design-guidelines/#begin-factory-name-with-make)